modules/apps/journal/journal-web/src/test/java/com/liferay/journal/web/internal/search/EntriesCheckerTest.java

```diff
@@ -0,0 +1,156 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.journal.web.internal.search;
+
+import com.liferay.journal.model.JournalArticle;
+import com.liferay.journal.model.JournalFolder;
+import com.liferay.journal.service.JournalArticleLocalServiceUtil;
+import com.liferay.journal.service.JournalFolderLocalServiceUtil;
+import com.liferay.journal.web.internal.security.permission.resource.JournalArticlePermission;
+import com.liferay.portal.kernel.language.LanguageUtil;
+import com.liferay.portal.kernel.portlet.LiferayPortletRequest;
+import com.liferay.portal.kernel.portlet.LiferayPortletResponse;
+import com.liferay.portal.kernel.security.permission.PermissionChecker;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.WebKeys;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import jakarta.servlet.http.HttpServletRequest;
+
+import java.util.Locale;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+/**
+ * @author Jan Brychta
+ */
+public class EntriesCheckerTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@AfterClass
+	public static void tearDownClass() {
+		_journalFolderLocalServiceUtilMockedStatic.close();
+		_journalArticleLocalServiceUtilMockedStatic.close();
+		_journalArticlePermissionMockedStatic.close();
+
+		_languageUtilMockedStatic.close();
+	}
+
+	@Test
+	public void testGetRowCheckBoxForNoJournalArticlePermissions() {
+		EntriesChecker entriesChecker = new EntriesChecker(
+			_getLiferayPortletRequest(),
+			Mockito.mock(LiferayPortletResponse.class));
+
+		JournalArticle mockArticle = Mockito.mock(JournalArticle.class);
+		JournalFolder mockJournalFolder = Mockito.mock(JournalFolder.class);
```

Comment:
Here you're mixing styles (article vs journalArticle, folder vs journalFolder). Pick one and stick with it (journalArticle / journalFolder is probably safer).